src/components/CartCounter/styled.js

```diff
@@ -1,8 +1,22 @@
-import styled from 'styled-components';
+import styled, {css} from 'styled-components';
 const StyledCartCounter = styled.span`
 	margin: 1rem;
 	padding: 0.6rem;
 	color: whitesmoke;
 	font-weight: bold;
+	${({variant}) =>
+		variant === 'counterHome' &&
+		css`
+			display: inline-block;
+			margin: 0rem;
+			padding: 0rem;
+			width: 20px;
+			line-height: 20px;
+			color: whitesmoke;
+			font-size: small;
+			background-color: lightsteelblue;
+			border-radius: 50%;
+			text-align: center;
+		`}
```

Comment:
Another nitty comment 🙂

So far you have only used this styled component element only once. 
For this reason, one variant is perhaps a little overdosed for now.